### DISCUSSION TITLE
Support for HifiBerry DIgi 2 Pro

### DISCUSSION THREAD

**Discussion by Argo16x** [rern/rAudio#979] [Feature Requests]
Today I got my Hifiberry Digi 2 Pro (2.2) to get rid of the USB problems with Technics SU-G700M2. I didn't check if it was supported by rAudio but as it is substantially based on same chip as Digi Pro +, I thought it could work. It doesn't. The card is detected but I had no sound output, whatever Digi driver I tried to use. I temporarily shiftet to HiFiBerry OS (it works) but I hope to see rAudio support for Digi 2 Pro soon.

Thank you.
Simone

**Comment by Argo16x** [comment]
Uhm, I think we can close this request. Actually disabling Audio I2S in System configuration and setting hifiberry-digi in Player configuration made things work together. I leave it here irf anyone else incur in my same "error".

Simone

**Reply by rern** [reply]
Please post the output of:
```sh
aplay -l
amixer scontrols
amixer scontents
```
It will be set automatically with next update.
